2024/migrant-issue-staging-bar/index.html

```diff
@@ -0,0 +1,44 @@
+<html lang="en">
+    <head>
+        <title>
+            Total Migrants in Staging Areas from June 2023 to February 2024
+        </title>
+        <!-- PLACE ALL REPOSITORY CSS FILES IN A BRACKET LIKE THIS -->
+        <!-- build:css inline -->
+        <link rel="stylesheet" href="../../static/main.css" />
+        <!-- /build -->
+    </head>
+
+    <body>
+        <div id="chart-div"></div>
+
+        <!-- EXTERNAL JS FILES -->
+        <!-- YOU LIKELY DO NOT NEED ALL OF THESE -->
+        <!-- IF YOU AREN'T USING AN IMPORT, IT'S RECOMMENDED THAT YOU REMOVE IT -->
+        <script
+            src="https://code.highcharts.com/11.0.0/highcharts.js"
+            type="text/javascript"
+        ></script>
+        <script
+            src="https://code.highcharts.com/11.0.0/modules/exporting.js"
+            type="text/javascript"
+        ></script>
+        <script
+            src="https://code.highcharts.com/11.0.0/modules/export-data.js"
+            type="text/javascript"
+        ></script>
+        <script
```

Comment:
Since you aren't importing the CSV for either of these, you should also be able to remove the data module for both of these, yeah?